### PR TITLE
Ui3

### PR DESCRIPTION
**Pull request by heemo521**

### CODE REVIEW

File: a2o.sql [hunk 1]
```diff
@@ -0,0 +1,28 @@
+-- DROP DATABASE CardGame;
+-- CREATE DATABASE CardGame;
+\c cardgame;
+-- CREATE SCHEMA cardgame;
+
```
Comment (codingavatar) [inline a2o.sql]: do we need another sql file?

**Review by codingavatar — APPROVED** [review]
Very clean and neat!

File: src/UI/AnimationCard.js [hunk 2]
```diff
@@ -21,9 +21,11 @@ export default function AnimationCard({ card, i }) {
     transform: "rotate3d(1, 1, 1, -45deg) scale(1.5)",
     cursor: "pointer",
     display: "inline-block",
-    animation: !animatestate
-      ? "rotate-in-diag-1 1s cubic-bezier(0.250, 0.460, 0.450, 0.940) both"
-      : `${rotateOut} 0.6s cubic-bezier(0.550, 0.085, 0.680, 0.530) both`,
+    animation:
```
Comment (carolinepeake) [inline src/UI/AnimationCard.js:24]: you meant to make this switched right?  the linter didn't switch it on you?

File: src/features/CreateGame.js [hunk 3]
```diff
@@ -30,14 +29,20 @@ export function CreateGame() {
...
   const { name, options, customCards } = CreateGameState;
 
+  useEffect(() => {
+    // Note for custom cards selection
+    // This useEffect function will run every time the customCards state changes
+    // This is because the customCards state is added to the dependency array for this useEffect function
+    // Do some conditional logic here and it should be good. And make sure the state is updated onChange of the checkbox
+
+    axios
+      .get("http://3.101.13.217:45000/cards/prompt")
```
Comment (carolinepeake) [inline src/features/CreateGame.js:39]: FYI, you changed the ip to localhost on line 10 of src/UI/Hero.js.  Don't know which one you want or if you want it to be consistent?

**Review by carolinepeake — APPROVED** [review]

**Pull request merged**